### PR TITLE
Minor performance and usability updates

### PR DESCRIPTION
**Pull request by NottyCode**
Update to prime the JVM class cache and liberty workarea for better startuptime.

Also symlink from the standard logs location to the docker one.

### CODE REVIEW

**Comment by arthurdm** [comment]
thanks @NottyCode.  Are the non-Kernel images doing a second start/stop?

**Comment by arthurdm** [comment]
In the `javaee7` images, I wonder if the start should be done via `docker-server`, so that the server doesn't start with SSL related error messages (such as `CWPKI0817A: The default SSL configuration expects a <keyStore> element ..`), and we also avoid the performance hit of creating the keystore during `docker run`?

**Comment by NottyCode** [comment]
@arthurdm The kernel images start the server with jsp-2.3 configured so I've had it do the same thing because it produces faster startup for the kernel image.

If I move the start/stop to docker-server then it will be run on first startup which would defeat the point of running it. I could update to remove any log files generated by the start stop which would mean no one would see any keystore 'error' messages.

**Comment by ewhoch** [comment]
Where is it grabbing the kernel image from? The new one that OL is starting to emit now? If so that explains the jsp-2.3 bit. It's actually a bit more work to make a new workspace where I can swap in new default templates and I wanted to get something done for the install teams to take a look at.

**Comment by NottyCode** [comment]
@ewhoch it is grabbing the open liberty image from maven central. There is only one, but the kernel image in docker creates a server using the defaultServer template which is jsp-2.3. It isn't quite kernel, but I don't know if I can change it having shipped it like this.

File: release/microProfile1/java8/ibmjava/Dockerfile [hunk 1]
```diff
@@ -2,3 +2,4 @@ FROM open-liberty:kernel-java8-ibm
 
 COPY server.xml /config/
 
+RUN /opt/ol/wlp/bin/server start && /opt/ol/wlp/bin/server stop && rm -rf /output/resources/security/
```
Comment (ewhoch) [inline release/microProfile1/java8/ibmjava/Dockerfile]: I'm not super familiar with the whole docker image, so I'm assuming  /output/resources/security simply gets populated with some data that we'd like to discard.

Comment (NottyCode) [reply release/microProfile1/java8/ibmjava/Dockerfile]: Yes, it contains ltpa keys and may contain ssl certificates.

**Pull request merged**